README.md

```diff
@@ -54,7 +54,7 @@ The steps are:
 2. Get the source code on to your machine via git.
 
     ```shell
-    git clone https://github.com/Cloud-CV/EvalAI.git evalai && cd evalai
+    git clone https://github.com/Cloud-CV/EvalAI.git && cd evalai
```

Comment:
It should be `cd EvailAI`. I just cloned it again and this is the folder name that it gave.